common/types.go

```diff
@@ -476,6 +476,7 @@ type WebhookVerificationParameters struct {
 	URL          string
 	ClientSecret string
 	Method       string
+	MatcherRefs  map[string]any // optional, depends on the provider
```

Comment:
can you explain what this is in a comment? Our types should be well documented, especially when the field name doesn't obviously communicate what it is.